src/main/proto/openrtb-xapi.proto

```diff
@@ -0,0 +1,565 @@
+//
```

Comment:
This should not be a part of repository, as proto files are downloaded at build time.